.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
           - rootfs-initrd
           - shim-v2
           - tdvf
+          - ovmf
```

Comment:
@fitzthum, we're trying to keep those alphabetically sorted. Mind to move this entry after nydus and before qemu ?